Makefile

```diff
@@ -47,13 +47,16 @@ endif
 REVISION=$(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .dirty; fi)
 RELEASE=${PROG}-$(VERSION).${GOOS}-${GOARCH}
 
-ifdef BUILDTAGS
```

Comment:
This logic was copied from containerd's Makefile so I don't even get what is was doing in the first place really.